argilla/src/argilla/settings/_resource.py

```diff
@@ -262,11 +262,19 @@ def from_json(cls, path: Union[Path, str]) -> "Settings":
 
     @classmethod
     def from_hub(
-        cls, repo_id: str, feature_mapping: Optional[Dict[str, Literal["question", "field", "metadata"]]] = None
+        cls,
+        repo_id: str,
+        feature_mapping: Optional[Dict[str, Literal["question", "field", "metadata"]]] = None,
+        **kwargs,
```

Comment:
I think is clearer if we make explicit the subset argument. Also, is more aligned to the Dataset.from_hub method definition 

```suggestion
        cls,
        repo_id: str,
        subset: Optional[str] = None,
        feature_mapping: Optional[Dict[str, Literal["question", "field", "metadata"]]] = None,
        **kwargs,
```